README.md

```diff
@@ -135,6 +157,33 @@ setupController: function(controller, model) {
   controller.set('products', model);
 }
 ```
+### event hooks
+
+ember-infinity provides a route mixin that enables us to hook into the following events:
+
+* **infinityModelUpdated**
+
+This gets fired up when data is loaded into the model.
```

Comment:
I think the word "fired up" is a little ambigious.  Can we change this to:

---

Triggered on the route whenever new objects are pushed into the infinityModel.

Args:
- totalPages